hibernate-core/src/test/java/org/hibernate/test/typeoverride/TypeOverrideTest.java

```diff
@@ -47,7 +39,13 @@ public void testStandardBasicSqlTypeDescriptor() {
 		assertSame( IntegerTypeDescriptor.INSTANCE, remapSqlTypeDescriptor( IntegerTypeDescriptor.INSTANCE ) );
 
 		// A few dialects explicitly override BlobTypeDescriptor.DEFAULT
-		if ( PostgreSQL81Dialect.class.isInstance( getDialect() ) || PostgreSQLDialect.class.isInstance( getDialect() ) )  {
+		if ( CockroachDB1920Dialect.class.isInstance( getDialect() ))  {
+			assertSame(
```

Comment:
It seems previously somewhere you use 'assertEquals' and here you are relying on references being the same. Are you sure 'assertSame' is the correct one, btw?